src/test/regress/sql/explain_format.sql

```diff
@@ -9,13 +9,13 @@
 -- m/^Settings:.*/
 -- s/^Settings:[,\s]*/Settings: /
 -- m/^Settings:.*/
--- s/,?\s*optimizer\w*\s*=\s*'off'//g
+-- s/,?\s*optimizer\w*\s*=\s*'on'//g
 -- end_matchsubs
 
 -- ignore variable JIT gucs which can be shown when SETTINGS=ON
 -- start_matchignore
 -- m/^\s+jit\w*:/
--- m/\s*optimizer:\s*"off"/
+-- m/\s*optimizer:\s*"on"/
```

Comment:
Why are these matchignores needed? Don't we catch everything with matchsubs above?